<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DodajPodjetje</class>
 <widget class="QDialog" name="DodajPodjetje">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>632</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Serif</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>true</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="18" column="1" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">gridline-color: rgb(176, 176, 176);
QHeaderView::section{padding:7px 15px}</string>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>St.</string>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Seznam strank</string>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Po≈°ta:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_posta</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_posta">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="Line" name="line_3">
     <property name="font">
      <font>
       <family>Serif</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dodaj stranko</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Naslov podjetja:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_naslov</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_kNaziv">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_naslov">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_ddv">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID DDV:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_ddv</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Celoten naziv:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_cNaziv</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_cNaziv">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Kratek naziv:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_kNaziv</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Elektronski naslov:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_email</cstring>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </item>
   <item row="20" column="1" colspan="2">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <family>Serif</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QPushButton" name="pushButton_dodajPodjetje">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Dodaj stranko</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QPushButton" name="pushButton_popraviPodatke">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Popravi podatke</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="2">
    <widget class="Line" name="line_4">
     <property name="font">
      <font>
       <family>Serif</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <widget class="QPushButton" name="pushButton_izhod">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Izhod</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="Line" name="line_2">
     <property name="font">
      <font>
       <family>Serif</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_kNaziv</tabstop>
  <tabstop>lineEdit_cNaziv</tabstop>
  <tabstop>lineEdit_naslov</tabstop>
  <tabstop>lineEdit_posta</tabstop>
  <tabstop>lineEdit_ddv</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>pushButton_popraviPodatke</tabstop>
  <tabstop>pushButton_dodajPodjetje</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>pushButton_izhod</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_izhod</sender>
   <signal>clicked()</signal>
   <receiver>DodajPodjetje</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
